triggers/incident-triggered/README.md

```diff
@@ -0,0 +1,52 @@
+# victorops-trigger-incident-triggered
+
+This [VictorOps](https://victorops.com) trigger fires when a new incident is created. 
+
+## Data Emitted 
+
+| Name              | Data type | Description |
+|-------------------|-----------|-------------|
+| `incident`        | string    | id of the incident | 
+| `webhook_payload` | mapping   | key-value pairs from the webhook payload |
+
+## Example Trigger
```

Comment:
If the example is in the step/trigger YAML, it no longer needs to be in the README.